releasenotes.rb

```diff
@@ -48,6 +48,16 @@ class ReleaseNotes < Thor
 
 ## Non-Customer Facing Changes
 
+## Special Deploy Requirements
+
+<% if commits.select(&:special_deploy_requirements).empty? %>
```

Comment:
I feel like the release notes are to communicate what changed about the software, the release requirements don't do that. so i think it's just confusing, especially for the non-developers. I guess it's more a question of who the release notes are for, which I think is John/David/Ping, etc.

If we have a special header/footer to mark the section, then we would be able to pull just that part. For example: 

```
***** SPECIAL RELEASE REQUIREMENTS *****
blah blah blah
***************************
```